src/test/java/io/axoniq/axonserver/connector/event/EventHandlingTest.java

```diff
@@ -143,8 +143,12 @@ void testCancelAggregateStream() throws InterruptedException {
 
         AggregateEventStream stream = eventChannel.openAggregateStream("aggregate1");
         stream.cancel();
-        while (stream.hasNext()) {
-            Assertions.assertNotEquals(199, stream.next().getAggregateSequenceNumber());
+        try {
+            while (stream.hasNext()) {
+                Assertions.assertNotEquals(199, stream.next().getAggregateSequenceNumber());
+            }
+        } catch (StreamClosedException e) {
```

Comment:
I'd almost assert the fact this exception is thrown with an `assertThrows` instead of silently swallowing it. However, this might not be the right test class for that.